website/docs/reference/resource-configs/hive-configs.md

```diff
@@ -0,0 +1,52 @@
+---
+title: "Apache Hive configurations"
+id: "hive-configs"
+---
+
+## Configuring tables
+
+When materializing a model as `table`, you may include several optional configs that are specific to the dbt-hive plugin, in addition to the standard [model configs](model-configs).
+
+| Option  | Description                                        | Required?               | Example                  |
+|---------|----------------------------------------------------|-------------------------|--------------------------|
+| partition_by | partition by a column, typically a directory per partition is created | No | partition_by=['name'] |
+| clustered_by | second level division of a partitioned column  | No | clustered_by=['age'] |
+| file_format | underlying storage format of the table | No | file_format='PARQUET' |
```

Comment:
is it true that you can use `parquet` with Hive? what other file formats are supported?

Reply:
text,csv,orc,avro and parquet are common one's. [Supported formats](https://cwiki.apache.org/confluence/display/Hive/FileFormats)